deploy/overlays/stanford-staging/kustomization.yaml

```diff
@@ -0,0 +1,54 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+nameSuffix: -stanford-staging
+namespace: website
+
+resources:
+- ../../base
+
+configMapGenerator:
+- behavior: create
+  literals:
+  - flaskEnv=stanford-staging
+  - secretProject=datcom-stanford-staging
+  name: website-configmap
+- behavior: create
+  literals:
+  - mixerProject=datcom-stanford-staging
+  - serviceName=website-esp.endpoints.datcom-stanford-staging.cloud.goog
+  name: mixer-configmap
+- behavior: merge
+  files:
+  - custom_bigtable_info.yaml
+  name: store-configmap
+
+patchesStrategicMerge:
+- |-
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: website-app
+  spec:
+    replicas: 6
+images:
+- name: gcr.io/datcom-ci/datacommons-mixer
+  newName: gcr.io/datcom-ci/datacommons-mixer
+  newTag: f0a283b
+- name: gcr.io/datcom-ci/datacommons-website
+  newName: gcr.io/datcom-ci/datacommons-website
+  newTag: b943707
```

Comment:
Please remove these. This is modified after a deploy.

Reply:
Ah yes. Thanks.